test/example.test.js

```diff
@@ -2,8 +2,12 @@ import { getRandomThrow, userWins } from '../get-random-throw.js';
 const test = QUnit.test;
 
 
-test('test should return true if a string is generated.', (expect) => {
-    const actual = getRandomThrow();
-    expect.equal(typeof actual, 'string');
+test('test should return true when paper beats rock.', (expect) => {
+    const actual = userWins('rock', 'paper');
+    const expected = true;
+
+    expect.equal(actual, expected)
 });
 
```

Comment:
Also you can test all the outcomes! (you don't have to do this but you can and prove your functions works flawlessly).